allennlp/predictors/open_information_extraction.py

```diff
@@ -60,6 +60,72 @@ def make_oie_string(tokens: List[Token], tags: List[str]) -> str:
 
     return " ".join(frame)
 
+def get_predicate_indices(tags: List[str]) -> List[int]:
+    """
+    Return the word indices of a predicate in BIO tags.
+    """
+    return [ind for (ind, tag)
```

Comment:
nit: you don't have to use parens here

another nit: you could probably fit this on one line (and if you can't, the `in` is a weird place to split it)